### PR TITLE
ep_mypads does not use ep_ldapauth

### PR DESCRIPTION
**Pull request by Dan33l**
#### Pull Request (PR) description

The helper about plugin `ep_mypads` uses the plugin `ep_ldapauth` settings.

But this is a bug. 

When the plugin `ep_mypads` is installed, it install the npm module `ldapauth-fork` and use it to bind with ldap server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ List of all plugins is avalable at https://static.etherpad.org/plugins.html
 
 If the plugin is not supported, it will be installed but whitout configuration.
 
-Exemple :
+Exemples :
```
Comment (bastelfreak) [inline README.md]:
```suggestion
Examples :
```

File: templates/plugins/ep_mypads.epp [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
 "ldap" : <%= to_json_pretty($etherpad::plugins::ep_mypads::_real_mypads_options) %> },
+
+/*When LDAP is configured, you add one Etherpad administrator. To see more, read the documentation.*/
+"users": {
+  "<%= $etherpad::ep_local_admin_login -%>": {
+    "password": "<%= $etherpad::ep_local_admin_pwd -%>",
+    "is_admin": true
+  }
+},
+
```
Comment (bastelfreak) [inline templates/plugins/ep_mypads.epp]: Please don't add the empty newline.

File: types/mypads.pp [hunk 3]
```diff
@@ -3,16 +3,11 @@
   {
     Optional['url']             => String,
     Optional['bindDN']          => String,
-    Optional['bindCredentials'] => String,
+    Optional['bindCredentials'] => String[1],
     Optional['searchBase']      => String,
     Optional['searchFilter']    => String,
     Optional['tlsOptions']      => Boolean,
     Optional['properties']      => Hash[String, String],
-    Optional['login']           => String,
```
Comment (bastelfreak) [inline types/mypads.pp]: were those fields never used or didn't it work? I'm not sure if this is a bugfix or a backwards-incompatible change.

Comment (Dan33l) [reply types/mypads.pp]: Accordingly with doc it looks that that are into the key `properties`. And so they are not at the top level of the structure.

https://framagit.org/framasoft/Etherpad/ep_mypads/wikis/use-ldap-authentication

**Pull request merged**